src/server/pfs/server/driver.go

```diff
@@ -1173,6 +1175,37 @@ func (d *driver) makeCommit(
 		}
 	}
 
+	// Set newCommitInfo.Started and possibly newCommitInfo.Finished. Enforce:
+	// 1) 'started' and 'newCommitInfo.Started' must be set
+	// 2) 'started' <= 'finished'
+	switch {
+	case started.IsZero() && finished.IsZero():
+		// set 'started' to Now() && leave 'finished' unset (open commit)
+		started = time.Now()
+	case started.IsZero() && !finished.IsZero():
+		// set 'started' to 'finished'
+		started = finished
+	case !started.IsZero() && finished.IsZero():
+		if now := time.Now(); now.Before(started) {
```

Comment:
What conditions would this ever happen where started is before now? If it's only in errors, would we want to fail loudly instead of silently modifying started?

Reply:
The main case where I could see this happening is time skew, but I see your point. My feeling is that this shouldn't derail a `Restore()` operation (which returning an error would do), but I did change this to log a warning if encountered